packages/background/src/keyring-keystone/service.ts

```diff
@@ -31,6 +31,7 @@ export class KeyRingKeystoneService {
         device: multiAccounts.device,
         deviceId: multiAccounts.deviceId,
         keys,
+        connectionType: multiAccounts.connectionType,
```

Comment:
Please make sure to additionally store bip44Path here. If there is bip44Path under insensitive, the bip44 path will be displayed to the user on the wallet select page. In the case of USB, it seems that the bip44 path can be used, but users often forget the path, so it needs to be displayed. See the implementation of keyring-mnemonic.

Reply:
added